docs/troubleshooting-guide.md

```diff
@@ -0,0 +1,87 @@
+# OPCT - Troubleshooting Guide
+
+- [Certification Failures](#review)
+- [Troubleshooting](#review-troubleshooting)
+    - [Review Results Archive](#review-archive)
+    - [Do I Need a Dedicated Test Environment](#review-needed-dedicated)
+    - [Cluster Failures](#review-cluster-failures)
+
+## Certification Tests Failures <a name="review"></a>
+
+Under any type of certification test failure, it is recommended to recreate the cluster under test. The certification tests check cluster metrics and logs which are persisted and this could impact subsequent certification tests.
+
+If you already know the reason for a test failure then resolve the problem and re-run the provider certification tool again so a new certification archive is created.
+
+If you are not sure why you have failed tests or if some of the tests fail intermittently, proceed with the troubleshooting steps below.
+
+> Note: When running the `preview` release of the certification tool, it's expected to have failed tests reported on the archive, we are working to improve the accuracy. If you are sure the failed tests reported on the archive are not related to your environment, feel free to contact your Red Hat partner to share the feedback.
```

Comment:
Since this documentation is going to be a part of the GA release, do you think we should still include the types of messages?

Reply:
No. It's like a call for feedback and can be written more directly

Reply:
Change to:

> > Note: The certification tool is in constant development, and due to the dynamic of the e2e test, you may experience failed tests reported on the archive that could be a flake, we are working to improve the accuracy of the reports. If you are sure the failed tests reported on the archive are not related to your environment, feel free to contact your Red Hat partner to share the feedback.